toxic-review-warning/src/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Toxic review warning</title>
+    <link rel="stylesheet" href="./styles/style-core.css" />
+    <link rel="stylesheet" href="./styles/style-ai.css" />
+    <link
+      rel="icon"
+      href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>☢️</text></svg>"
+    />
+  </head>
+  <body>
+    <main>
+      <div>
+        <h1>Toxic review warning ☢️</h1>
+        <div id="toxicityStatusTemp"></div>
+        <div class="img-wrapper">
+          <img src="./book.png" alt="" width="200">
+        </div>
+        <div>
+          <div id="modelStatusWrapper">
+            <div id="modelStatus"></div>
+          </div>
+          <div class="input-area">
+            <div>
+              <label for="reviewInputEl">Product review</label>
+            </div>
+            <div class="input-wrapper">
+              <textarea name="reviewInputEl" id="reviewInputEl" oninput="handleUserInputChange()">I freaking hate this book</textarea>
+            </div>
+            <div class="button-wrapper">
+              <div id="aiWrapper">
+                <div class="ai-effect-wrapper">
+                  <div id="aiEffect"></div>
+                </div>
+                <div id="toxicityAssessmentEl"></div>
+              </div>
+              <button onclick="simulatePostReview()">
+                Post
+              </button>
+            </div>
+          </div>
```

Comment:
Wrap this in a `<form>` and make the `<button type=submit>`. Remember to listen for `FormElement.onsubmit` and cancel the default behavior.

Reply:
Done 👍